content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -0,0 +1,139 @@
+entered, click :guilabel:`Send by Email`. This can be useful when creating a call for tender,
+because vendors can confirm if their past prices still hold today, which can help companies choose
+the best offers for them.
+
+Create alternatives to a :abbr:`RFQ (request for quotation)`
+------------------------------------------------------------
+
+Once a purchase order is created, alternative quotations can be created and sent to vendors to
+compare for the best deal later. To do this, click the :guilabel:`Alternatives` tab, then click
+:guilabel:`Create Alternative`. This will cause a popup to appear.
+
+.. image:: calls_for_tenders/calls-for-tenders-alternatives-popup.png
+   :align: center
+   :alt: Calls for tenders popup to create alternative quotation.
+
+From the popup, click the drop-down menu next to :guilabel:`Vendor`, and select a vendor for the
+alternative :abbr:`PO (purchase order)`. Next to this, there is a :guilabel:`Copy Products` checkbox
+that is selected by default. When selected, the product quantities of the original
+:abbr:`PO (purchase order)` will be copied to the alternative. Once finished, click
+:guilabel:`Create Alternative`. This will create and navigate to a new purchase order.
+
+Once again, click :guilabel:`Send by Email`. Next, create a second alternative by
+clicking the :guilabel:`Alternatives` tab, and click :guilabel:`Create Alternative`. When the popup
+appears again, choose a different vendor from the drop-down menu next to :guilabel:`Vendor`, and
+this time, *uncheck* the :guilabel:`Copy Products` checkbox. Click :guilabel:`Create Alternative`.
+
+.. tip::
+    If one or two alternative quotations need to be removed from the :guilabel:`Alternatives` tab,
+    they can be individually removed by clicking on the black "X" icon at the end of their row.
+
+This will create a new purchase order, but since the product quantities of the original
+:abbr:`PO (purchase order)` were not copied over, new products will need to be added to the product
+lines. Add the products, then click :guilabel:`Send by Email`.
+
+From this newest purchase order, click the :guilabel:`Alternatives` tab. From here, all three
+purchase orders can be seen, inluding which vendors they were sent to, the expected arrival of the
+products if confirmed, and the status of the order. Before comparing any product lines, click
+:guilabel:`New` at the top of the page. This will create a brand new quotation. Fill out the
+information on this new quotation, and click :guilabel:`Send by Email`. Then, click the
+:guilabel:`Alternatives` tab once more. On this new :abbr:`PO (purchase order)`, there are no other
+orders linked. To link this order with the alternatives, click :guilabel:`Link to Existing RfQ`.
+
+.. image:: calls_for_tenders/calls-for-tenders-link-existing-rfq.png
+   :align: center
+   :alt: Popup to link new quotation to existing RFQs.
+
+This will cause a popup to appear. From the popup, select the three purchase orders created
```

Comment:
This causes a pop-up to appear.

(you can use popup or pop-up - don't think that matters. But, whichever you choose, make sure it stays uniform throughout the documentation)